src/components/TokenBar.js

```diff
@@ -141,9 +140,9 @@ class TokenBar extends React.Component {
 
     const renderUnknownTokens = () => {
       return (
-        <div title={`${unknownTokens} unknown ${helpers.plural(unknownTokens, 'token', 'tokens')}`} className={`d-flex align-items-center icon-wrapper ${this.state.opened ? 'justify-content-start' : 'justify-content-center'}`} onClick={this.unknownClicked}>
+        <div title={`${unknownTokens} unknown ${hathorLib.helpers.plural(unknownTokens, 'token', 'tokens')}`} className={`d-flex align-items-center icon-wrapper ${this.state.opened ? 'justify-content-start' : 'justify-content-center'}`} onClick={this.unknownClicked}>
```

Comment:
I don't feel that `plural` should be in hathorLib.

Reply:
Humm, I agree that shouldn't be in the lib, I just thought that we use it in both desktop and mobile wallet, then we would need to recreate it there.

Reply:
Ok. Make sense.